### PR TITLE
Feat/support saving setup 11

### PR DESCRIPTION
**Pull request by mrrobot16**

### CODE REVIEW

**Comment by mrrobot16** [comment]
This PR is for issue: https://github.com/dOrgTech/DAOcreator/issues/11

File: src/components/pages/DAOcreator/index.tsx [hunk 1]
```diff
@@ -36,14 +43,90 @@ interface Step {
...
+    } = this.state.daoCreatorSetup;
+    const daoCreatorState = {
+      config: {
+        daoName: orgName,
+        tokenName,
+        tokenSymbol
+      },
+      members: founders,
+      // TODO
+      // Need to figure out how to create the array of schemes.
```
Comment (mrrobot16) [inline src/components/pages/DAOcreator/index.tsx]: Here is were i am stuck.

**Comment by mrrobot16** [comment]
We still need to still style it :) 
Screen Shot 2019-10-30 at 4 26 01 PM

**Comment by mrrobot16** [comment]
Its working now. Needs minor styling improvement.

**Comment by dOrgJelli** [comment]
@mrrobot16 @namesty the "save state" + "restore schemes" logic was unneeded as deseralization is already implemented in the lib + state + form layers (see `fromJSON`, `fromDAOMigrationParams`, and `fromState`).

Please look at this diff to understand the simpler approach that could have been taken: https://github.com/dOrgTech/DAOcreator/pull/222/commits/d972a7ef4e7f94d716d28d86ee217c094c9e1cfb

**Pull request merged**